icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java

```diff
@@ -90,7 +354,179 @@ public String getSubtype() {
         return subType;
     }
 
+    /**
+     * Gets the CLDR Sequence Unit Identifier for this MeasureUnit, as defined in UTS 35.
```

Comment:
I don't think we call it a "Sequence Unit Identifier" any more.  CC @hugovdm

Reply:
Just call it a CLDR Unit Identifier.  Also fix C++.

Reply:
Do you want the C++ fix in the same pull request or in another one?

Reply:
Done for Java

Reply:
https://github.com/icu-units/icu/issues/77 

for C++